.github/workflows/deployment.yml

```diff
@@ -325,27 +329,27 @@ jobs:
         env:
           BUILDJABREFPRIVATEKEY: ${{ secrets.buildJabRefPrivateKey }}
       - name: Download from GitHub workflow artifacts store (macOS)
-        if: (steps.checksecrets.outputs.secretspresent == 'YES') && (startsWith(github.ref, 'refs/tags/') || inputs.notarization == true)
```

Comment:
Do we always want to do notarization now?

Reply:
Please check line 318 - the whole notarization job is only started if we want notarization. Thus, the steps itself do not need check again.

![image](https://github.com/JabRef/jabref/assets/1366654/a7c01476-f06c-4064-ad98-a3d825b51241)

```yaml
    if: ${{ startsWith(github.ref, 'refs/tags/') || inputs.notarization == true }}
```